modules/nw-networkpolicy-audit-concept.adoc

```diff
@@ -26,7 +33,78 @@ metadata:
       }
 ----
 
-The logging format is compatible with syslog as defined by RFC5424. The syslog facility is configurable and defaults to `local0`. An example log entry might resemble the following:
+You can configure log file parameters for the OVN-Kubernetes network plugin. The following example lists these parameters and their default values:
+
+.Example Go file that sets log file parameters for the OVN-Kubernetes network plugin
+[source,go]
+----
+	Logging = LoggingConfig {
+		File:                "", <1>
+		CNIFile:             "",
+		LibovsdbFile:        "",
+		Level:               4,
+		LogFileMaxSize:      100, <2>
+		LogFileMaxBackups:   5,
+		LogFileMaxAge:       5, <3>
+		ACLLoggingRateLimit: 20,
+  	}
+----
+<1> Do not specific the path to a specific file.
```

Comment:
Need to fix description,.

Reply:
The Go file is now removed so this line no longer exists.